### PR TITLE
Nfithirdparty

### PR DESCRIPTION
**Pull request by ArchanaWable**
## Description
HPHX-12379-Added a space between : and MyBarcodeScanner, 
HPHX-12380-Incorrect Instructions in cards are incorrect, 
HPHX-12381-Incorrect Instructions in cards are incorrect/ Functional, 
HPHX-12382-Corrected frameworks name: android.util.jar, 
HPHX-12383-Add a space between : and MyBarcodeScanner, 
HPHX-12386-Moved the card to a more suitable position, 
HPHX-12387 - Change the description for Card 12 : step1 Enter Invoke Function in the sear bar and select Invoke Function under the Function section

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Screenshots FYI**

HPHX-12386-C12-before

HPHX-12386-C12-before

HPHX-12386-C12-after
Changed the position of the card on right side

HPHX-12386-C12-after

HPHX-12379-C8 Before

HPHX-12379-C8-before

HPHX-12379-C8 After

HPHX-12379-C8-after

HPHX-12387-C12 Before

HPHX-12387-C12-before

After

HPHX-12387-C12-after

HPHX-12383, 12380, 12381, 12382_C9 Before
HPHX-12383, 12380, 12381, 12382_C9-before

HPHX-12383, 12380, 12381, 12382_C9 After
HPHX-12383, 12380, 12381, 12382_C9-after

### CODE REVIEW

File: public/contents/6-advanced-concepts/tours.json [hunk 1]
```diff
@@ -48,6 +48,25 @@
       "kuid": "f9e5f0t5-a8a5-4arb-bfu1-50d392n4",
       "hikeVersion": "1.0",
       "weight": "2"
+    },
```
Comment (mandeepalluru) [inline public/contents/6-advanced-concepts/tours.json:51]: In tour.json, only change should be checksum for respective hike.
@AshokMamidiHcl please guide here

**Comment by mandeepalluru** [comment]
For HPHX-12387, I just wanted to check if the following mentioned in the ticket has been taken care of.
image

**Comment by mandeepalluru** [comment]
For HPHX-12386, may I know the new position of the card. Please post screenshots of before change and after change.

**Comment by mandeepalluru** [comment]
@ArchanaWable For all fixes, please post screenshots of before change and after change.

**Comment by mandeepalluru** [comment]
@AshokMamidiHcl , request you to do one quick round of review post the requested changes.

**Comment by AshokMamidiHcl** [comment]
Checksum should be updated. Close this PR

**Comment by ArchanaWable** [comment]
The Checksum should only updated.

**Pull request closed without merging**